Source/Core/Core/Boot/Boot.cpp

```diff
@@ -300,19 +308,46 @@ bool CBoot::LoadMapFromFilename()
   return false;
 }
 
+void RAMOverrideForIPLMemoryValues(std::string& data, const u32 ipl_hash)
```

Comment:
```suggestion
static void RAMOverrideForIPLMemoryValues(std::string& data, const u32 ipl_hash)
```

Otherwise this results in a -Wmissing-declaration warning.